tracer/test/snapshots/AerospikeTests.SchemaV0.verified.txt

```diff
@@ -16,7 +16,6 @@
     },
     Metrics: {
       process_id: 0,
-      _dd.agent_psr: 1.0,
```

Comment:
To clarify, if we _were_ to add returning of sampling rates from the mock agent, that could make all this quite flaky right? Not an issue obviously, we can just not add them, just something for us to remember

Reply:
Yes, exactly.